pom.xml

```diff
@@ -12,7 +12,7 @@
   <version>1.36-SNAPSHOT</version>
 
   <properties>
-    <java.level>3</java.level>
+    <java.level>6</java.level>
```

Comment:
@oleg-nenashev haha maybe one more year? :-)
Well it looks @daniel-beck is happy with 6! So let's use this current (temporary?) mood before he change his mind :P